src/extension/options.js

```diff
@@ -36,6 +36,10 @@ function isValidGitHubURL(giturl) {
     && Object.keys(getGitHubSettings(giturl)).length === 3;
 }
 
+function isInvalidMountpoint(mountpoints) {
```

Comment:
I think I would turn this into `isValidMountpoint` (to align with `isValidHost` etc)